### ISSUE TITLE
Some of the T_N Giygas death minigames doesn't show two specific things in this engine.

### ISSUE THREAD

**Issue by FrankSP7ps** [ikemen-engine/Ikemen-GO#1476]
### Describe the bug

So Giygas (The_None) has many death minigames (minigames that requires to do specific actions to save you opponent, if you don't you will oponent will die and that would take as a "K.O"). One of them is save the char that you are controlling from being killed by one of the fatalities stages from Eternal Champions by executing quickly the buttons that appear on screen.

Normally it would show the char and if you fail it would show a body that got mutilated and stuff.

For some reason here in Ikemen go  doesn't show the char and the body when you failed.

### To Reproduce

Giygas is made by The_None, soo. The same page as always.
https://www.justnopoint.com/lbends/index.php?location=chars&page=1

Opponent can be anyone.

before to test make this little change on giygas .cns

1: go to [Statedef 3010]
2: then find this in giygas.cns
 
[State 0, ChangeState]
type = ChangeState
trigger1 = time = 80
value = ifelse(numtarget && !(target,ishelper), 3100 + (random%6*100) ,3015)
;value = ifelse(numtarget && !(target,ishelper), 3400 ,3015)
ctrl = 0

now change the value like this

value = ifelse(numtarget && !(target,ishelper), 3100 ,3015)

With that Giygas should only pick that one death minigame instead of waiting that Giygas pick the one that you want to see.

3: Pick giygas and choose any opponent
4: Use portal move by pressing Z
5: Get the opponent to the portal
6: Test the minigame, win or fail
7: Profit!

### Expected behavior

  The minigame should work wine, just doesn't show the opponent falling and after the machines killiing the opponent it doesn't show the body.

### Screenshots / Video

https://github.com/ikemen-engine/Ikemen-GO/assets/108790388/386784d4-1d3d-4d5a-b233-7f71fcf6e5fc

### Engine Version (or source code date)

v0.99.0

### Operating system

Windows

**Comment by potsmugen** [comment]
Weird. It works for me with any of its 4 def files.

![ikemen000](https://github.com/ikemen-engine/Ikemen-GO/assets/107247004/4559be15-1676-4753-a386-a269f7901826)

**Comment by FrankSP7ps** [comment]
Well that's weird.
I did a clean re install on him and well...
![ikemen134](https://github.com/ikemen-engine/Ikemen-GO/assets/108790388/fbe2c9fb-6221-460c-997b-99477c12c54a)
Yeah it works.

Hmm...It must have been something when I moved all the code from the 10ov.st to the .cns (Ikemen GO has that little bad problem that ignores the states that are the same no matter if I put it first in "st" or later "st1", bringing the only solution on move all the 1.0 or 1.1 stuff to the .cns).

I will do a recheck after do that.
